### ISSUE TITLE
Synchronizing names back-to-front

### ISSUE THREAD

**Issue by jadudm** [GSA-TTS/FAC#1967]
### Name sync

Data has many names in this application.

1. The workbooks have named ranges.
2. The intake model has names for data. Generally, these are expressed via the workbook named ranges (which are passed through from workbook names -> validation -> mapping -> JSON object in the intake DB), or via form->JSON. 
3. Those names are then referenced in cross-validations.
4. They are further referenced in generating workbooks for testing in the `workbooks_e2e` app. This app maps our workbook column names to the dissemination names, because it tests by inserting data into a workbook, and validating that the same data comes out at a given API endpoint and field name.

So, name changes might require us to look at:

1. The workbook named range, meaning the Jsonnet spec for the workbook.
2. The validations within Jsonnet, that might reference a workbook column name.
3. The mapping from workbook/form (or, UI) JSON to the intake model references these names. 
4. The model for the 2023 intake refers to those names.
5. Cross validations refer to the model names.
4. The workbooks_e2e contains mapping from named ranges to dissemination field names.
6. The dissemination model expresses the outgoing names.

The UI, for forms, might embed names as well, so making sure the UI/views for the forms are correct (if we touch those names) is a parallel pipeline to the workbooks.

### Process

1. Look at the dissemination model.
2. Determine if the name matches the workbook named range.
3. If not, consider renaming the dissemination. If we rename a dissemination model field, we must change `workbooks_e2e`,  `IntakeToDissemination`,  and the API view for `v1_0_0_beta`.
4. If we decide we have to change the workbook named range, then we have to check the validation schemas, the mapping code, the application model, any associated views (templates, UI), tests, data fixtures, cross-validation, and the list associated with dissemination (step 3).

If we change anything, we should have a spreadsheet that contains a checklist (columns) for each of those actions, so we explicitly track the work as a group as we do it. Even if we're pairing, we should track it as we do it. 

Below, we'll grow the checklist for each thing we rename. That is, if we rename a dissemination field, we should say: "Dissemination change: name_one -> name_two." Or, "Workbook change: name_one -> name_two." The tracking of everything else can be in an associated spreadsheet in GDrive. (Or, we can use a checklist here for each place we touch in that case.)

```[tasklist]
### Models
- [x] FederalAwards
- [x] Findings
- [x] Passthrough
- [x] Notes to SEFA
- [x] Corrective Action Plan Text
- [x] Secondary Auditors
- [x] General
```

-----------------------------------

```[tasklist]
### FederalAwards
- [x] Dissemination model
- [x] IntakeToDissemination
- [ ] Dissemination tests
- [ ] workbooks_e2e
- [ ] API views
```

```[tasklist]
### Findings
- [x] Dissemination model
- [x] IntakeToDissemination
- [ ] Dissemination tests
- [ ] workbooks_e2e
- [ ] API views
```

```[tasklist]
### Passthrough
- [x] Dissemination model
- [x] IntakeToDissemination
- [ ] Dissemination tests
- [ ] workbooks_e2e
- [ ] API views
```

```[tasklist]
### Notes to SEFA
- [x] Dissemination model
- [x] IntakeToDissemination
- [ ] Dissemination tests
- [ ] workbooks_e2e
- [ ] API views
```

```[tasklist]
### CapText
- [x] Dissemination model
- [x] IntakeToDissemination
- [ ] Dissemination tests
- [ ] workbooks_e2e
- [ ] API views
```

```[tasklist]
### SecondaryAuditors
- [x] Dissemination model
- [x] IntakeToDissemination
- [ ] Dissemination tests
- [ ] workbooks_e2e
- [ ] API views
```

```[tasklist]
### General
- [x] Dissemination model
- [x] IntakeToDissemination
- [ ] Dissemination tests
- [ ] workbooks_e2e
- [ ] API views
```

**Comment by jadudm** [comment]
I've removed the `Revisions` table. It reflects the design of the historic table, and we do not yet handle revisions. How we do, or do not, bring this forward will need to be the subject of discussion and design. So, for now, it feels better to yank it from the `IntakeToDissemination` code than to leave it there with `None` values.

**Comment by jadudm** [comment]
`Passthrough`, as a table/model, was completed as part of the `FederalAwards` work.

**Comment by danswick** [comment]
In a similar vein, I removed the `hist_` items from `SecondaryAuditors`. I believe we're trying to anticipate historical data intake, but I erred on the side of consistency for now since we aren't collecting that information and aren't sure how we'll handle historical data when the time comes. Should be easy to move them back if this was the wrong move!

Also noting there are a few `hist_` items in the `General` model in case we want to remove those as well.
